applovin_max/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Versions
 
+## x.x.x
+* Add support for BigoAds native ads.
```

Comment:
Same comment as React Native, but should we indicate what the actual change was too and that it only applies to Android? (a bit unclear from strictly reading the code)